lib/govspeak/presenters/image_presenter.rb

```diff
@@ -0,0 +1,25 @@
+module Govspeak
+  class ImagePresenter
+    attr_reader :image
+
+    def initialize(image)
+      @image = image
+    end
+
+    def url
+      image.respond_to?(:url) ? image.url : image[:url]
+    end
+
+    def alt_text
+      image.respond_to?(:alt_text) ? image.alt_text : image[:alt_text]
+    end
+
+    def caption
+      (image.respond_to?(:caption) ? image.caption : image[:caption]).to_s.strip.presence
+    end
+
+    def id
+      nil
```

Comment:
Does this method need to exist?

Reply:
This is called by the `render_image` method, so the presenter has to have the same interface. It is defined in AttachmentImagePresenter, and it seems reasonable that we might want to define it properly for images, someday i.e. the common interface seems reasonable.

Reply:
Ah gotcha 👍